Readme.md

```
@@ -10,4 +10,8 @@
 ```
 dotnet ef migrations add InitialCreate
 dotnet ef database update
-```
\ No newline at end of file
+```
+
+# Pour restaurer un backup
+
+```"D:\Program Files\PostgreSQL\18\bin\psql.exe" -U postgres -d BASE_DE_DONNEE -f "backup.sql"```
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)